src/lavinmq/vhost.cr

```diff
@@ -35,6 +37,7 @@ module LavinMQ
     @direct_reply_consumers = Hash(String, Client::Channel).new
     @shovels : ShovelStore?
     @upstreams : Federation::UpstreamStore?
+    @sessions : MQTT::SessionStore?
```

Comment:
Is this store really needed? Isn't the queue (session) enough?

Reply:
yes, just a rest from experimenting!

Reply:
The "session methods" in `Vhost` feels misplaced. Maybe the `SessionStore` should exist as a wrapper around `Vhost` and move the "session methods" to `SessionStore`. Then e.g. `SessionStore#create(client_id)` will check existence of queue etc and create if necessary?